comments/screenshot.py

```diff
@@ -44,12 +45,22 @@ def download_screenshots_of_reddit_posts(accepted_comments, url, video_directory
             cookie_file = open(
                 f"{os.getcwd()}/comments/cookie-light-mode.json", encoding="utf-8"
             )
-        cookies = json.load(cookie_file)
-        context.add_cookies(cookies)  # load preference cookies
+
         # Get the thread screenshot
         page = context.new_page()
+
+        page.goto('https://www.reddit.com/login')
+        page.type('#loginUsername', auth.praw_username)
+        page.type('#loginPassword', auth.praw_password)
+        page.click('button[type="submit"]')
+        page.wait_for_url('https://www.reddit.com/')
+
+        cookies = json.load(cookie_file)
+        context.add_cookies(cookies)  # load preference cookies
```

Comment:
This needs additional work done to it, looks like after adding in the Reddit Login it is not applying the Dark theme cookie anymore, needs more troubleshooting.